PLNPlugin.inc.php

```diff
@@ -237,8 +226,6 @@ public function getSetting($journalId, $settingName) {
 				break;
 			case 'pln_network':
 				return Config::getVar('lockss', 'pln_url', PLN_DEFAULT_NETWORK);
-			case 'pln_status_docs':
```

Comment:
Not used, anyway, this "documentation" is a bit outdated and is strictly technical.